app/services/foreman/plugin.rb

```diff
@@ -243,7 +243,10 @@ def pending_migrations
       pending_migrations = ActiveRecord::Migrator.new(:up, migration_paths).
         pending_migrations
 
-      pending_migrations.size > 0
+      return false if pending_migrations.size.zero?
+      Rails.logger.debug(
+        "Pending migrations: #{pending_migrations.map(&:name).join(', ')}")
```

Comment:
Not necessarily I guess - contents are not that important since all will run with db:migrate anyway